### PR TITLE
Chet 257 form section in panel

### PR DESCRIPTION
**Pull request by theghostwhoforks**
Creates a panel that hides the optional sections. All form sections can not be manually closed, should an admin desire to do so

Screenshot 2020-04-22 at 15 37 07

And here's what the form looks like with all the sections collapsed - By default, only the sections with optional in the name are collapsed.
Screenshot 2020-04-22 at 15 44 34

### CODE REVIEW

File: frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx [hunk 1]
```diff
@@ -53,8 +54,15 @@ const QuickstartFormContainer = styled.form`
...
-const QuickstartSubmitButton = styled(Button)`
-    margin-top: 10px;
+const PanelContainer = styled.div`
+    & span {
+        font-size: 1.4em;
+        font-weight: 400;
+    }
+`;
+
+const ButtonRow = styled.div`
```
Comment (theghostwhoforks) [inline frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx:64]: feedback from a previous PR

File: frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx [hunk 2]
```diff
@@ -130,16 +138,36 @@ const QuickstartForm = ({
                 <StackNameField />
                 {quickstartParamGroups.map(group => {
                     return (
-                        <FormSection key={group.groupLabel} title={group.groupLabel}>
-                            {group.parameters.map(parameter => {
-                                return createQuickstartFormField(parameter);
-                            })}
-                        </FormSection>
+                        <PanelContainer key={`${group.groupLabel}-panelContainer`}>
```
Comment (theghostwhoforks) [inline frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx:141]: the joy of wrapping thanks to `styled(Panel)` not being available 😞

**Comment by theghostwhoforks** [comment]
Updated input elements to have helper message underneath -

Screenshot 2020-04-22 at 16 52 21

**Comment by bpar476** [comment]
This looks a bit different from the design spec. In the design spec the other sections can't be collapsed at all. The cluster nodes section is also behind a collapse and with the other optional groups. Just wanted to check in if this is intentional

File: frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx (hunk 2 above)
Comment (bpar476) [reply frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx:141]: ☹️

File: frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx [hunk 3]
```diff
@@ -168,6 +184,7 @@ const buildQuickstartParams = (quickstartParamDoc: any): Array<QuickstartParamet
         const paramGroupLabel = Label;
         return {
             groupLabel: paramGroupLabel.default,
+            shouldExpandGroupOnLoad: !/optional/i.test(paramGroupLabel.default),
```
Comment (bpar476) [inline frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx:187]: While this is elegant, I feel like to achieve the desired design, we're going to need to hardcode the "optional" groups as they don't all have "optional" in the name. :(

Comment (theghostwhoforks) [reply frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx:187]: Can be done easily -> i'll generate the form element based on the type of group that's coming in

Comment (theghostwhoforks) [reply frontend/src/main/dc-migration-assistant-fe/components/aws/quickstart/QuickStartDeploy.tsx:187]: https://app.zeplin.io/project/5e9ea63418e4a27a9986919e/screen/5e9ea7cd137d2e518dfa8ddf

The latest designs require a fair amount of work as the mapping between the Cloudformation YAML and our UI is not 1:1

Parts of the DB sections are shown explicitly, while part of it is rendered in a collapsible section. I'll merge this in with the sections being collapsed for groups that have optional in the name for now

**Comment by theghostwhoforks** [comment]
> This looks a bit different from the design spec. In the design spec the other sections can't be collapsed at all. The cluster nodes section is also behind a collapse and with the other optional groups. Just wanted to check in if this is intentional


I've changed this to make only the optional sections collapse. This is still different from the design spec, but we can review this in the demo - I feel this is good, given the how the YAML defines the groups.

**Pull request merged**